lib/game_play.rb

```diff
@@ -13,29 +13,47 @@ class GamePlay
   def initialize(ui)
     @board = Board.new
     @ui = ui
+    @VALUE_MIN = 1
   end
 
   def board_size
-    @ui.board_size
-    selection = get_user_selection_between(3, 4)
+    default_size = 1
+
+    @ui.display_menu_board(default_size)
+    menu_size = @ui.menu_board_size
+    selection = get_user_value(@VALUE_MIN, menu_size, default_size)
+
+    selection += 2
```

Comment:
I expected a feedback about this one! I still don't find a better way.

Basically there is two size in my menu:

- 3x3 
- 4x4

To select the 3x3 board the user types 1 and for the 4x4 board he types 2. By adding 2 I get the size of the board!

Until I had a new board which increment by more than one 👎